File: pubsub/Sender.java
package pubsub;

import org.vertx.java.core.Handler;
import org.vertx.java.core.eventbus.Message;
import org.vertx.java.platform.Verticle;

public class Sender extends Verticle {

  public void start() {
    // Publish some news on the feed every second
    vertx.setPeriodic(1000, new Handler<Long>() {
      @Override
      public void handle(Long timerID) {
        System.out.println("sending from Java...");
        vertx.eventBus().publish("news-feed", "Java News " + new java.util.Date());
      }
    });
  }
}

File: pubsub/sender.js
var eb = require("vertx/event_bus");
var console = require("vertx/console");
var vertx = require("vertx")

vertx.setPeriodic(1000, function sendMessage() {
	console.log("sending from JS...");
  eb.publish('news-feed', 'JavaScript News');
})
